src/ucp/proto/proto_am.inl

```diff
@@ -70,10 +72,23 @@ ucs_status_t ucp_do_am_bcopy_multi(uct_pending_req_t *self, uint8_t am_id_first,
         packed_len = uct_ep_am_bcopy(uct_ep, am_id_last, pack_last, req, 0);
         UCS_PROFILE_REQUEST_EVENT_CHECK_STATUS(req, "am_bcopy_last", packed_len,
                                                packed_len);
-        return (packed_len < 0) ? packed_len : UCS_OK;
+        last = 1; //return (packed_len < 0) ? packed_len : UCS_OK;
```

Comment:
remove last variable by doing (if packed_len>=0) return UCS_OK here, and falling back to line 78 in case of error

Reply:
fixed